src/main/java/Advent/Day3.java

```diff
@@ -0,0 +1,34 @@
+package Advent;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Day3 {
+
+    public static long solveDay3(List input) {
```

Comment:
Don't throw away part 1 solution, no matter how conveninet it is to continue on top. Aim to support both. Ideally, aim to create part 2 upon part 1 solution.